doc/source/cookbook/simple_projection_methods.py

```diff
@@ -0,0 +1,13 @@
+import yt
+
+# Load the dataset.
+fname = "Shattering_V10_hdf5_plt_cnt_2101"
+ds = yt.load("/Users/ryanjsfx/SCP/ReceivedFiles/Shattering/sR1e5/" + fname)
```

Comment:
The path to the sample doesn't seem to exist in my repo and I didn't see it in git_ignore...probably I'm missing something simple. I guess I'll look around for it tomorrow.

Regarding conforming to code style standards, sorry probably I'm confused/wrong but I thought pre-commit is supposed to run automatically after I open a pull request (?)

I changed the filename back to what was used in the other cookbook script so hopefully that helps.

Reply:
Found the existing dataset here: https://yt-project.org/data/index.html
It's downloading but I'll test the new method with it later :)

Reply:
> Regarding conforming to code style standards, sorry probably I'm confused/wrong but I thought pre-commit is supposed to run automatically after I open a pull request (?)

Sorry I missed that. pre-commit can be installed locally (recommended) so you can validate it _as you commit_, or even after that with `pre-commit run --all-files` (this takes a couple minutes to run). This is the easiest way to work collectively with pre-commit. Otherwise, it's run in CI in a check-only way, and we need to invoke the CI bot manually via comments to run it again and fix stuff automatically. This workflow isn't ideal because not everything can be auto-fixed.

Reply:
Yup, got pre-commit installed locally on my laptop and it looks like it's running when do git commits now. It's not working on my work desktop though :( so I'll try to avoid using that machine.